mitiq/shadows/quantum_processing.py

```diff
@@ -82,44 +82,51 @@ def random_pauli_measurement(
     circuit: cirq.Circuit,
     n_total_measurements: int,
     executor: Callable[[cirq.Circuit], MeasurementResult],
-) -> Tuple[NDArray[Any], NDArray[Any]]:
-    r"""
-    Given a circuit, perform random Pauli measurements on the circuit and
-    return outcomes. The outcomes are represented as a string where a
-    z-basis measurement outcome of :math:`|0\rangle` corresponds to 1, and
-    :math:`|1\rangle` corresponds to -1.
+    qubits: Optional[List[cirq.Qid]] = None,
+) -> Tuple[List[str], List[str]]:
+    r"""This function performs random Pauli measurements on a given circuit and
+    returns the outcomes. These outcomes are represented as a tuple of two
+    lists of strings.
 
     Args:
-        circuit: Cirq circuit.
-        n_total_measurements: number of snapshots.
-        executor: A callable which runs a circuit and returns a single
+        circuit: A Cirq circuit.
+        n_total_measurements: The number of snapshots.
+        executor: A callable that runs a circuit and returns a single
             bitstring.
+        qubits: The qubits to be measured. If None, all qubits in the circuit
+            will be measured.
 
     Warning:
-        The ``executor`` must return a ``MeasurementResult``
-        for a single shot (i.e. a single bitstring).
+        The `executor` must return a `MeasurementResult` for a single shot,
+        i.e., a single bitstring.
```

Comment:
I think we want the double backticks around `executor` and `MeasurementResult`. With just one it renders as italic, not code.
<img width="680" alt="Screenshot 2023-08-02 at 7 41 50 AM" src="https://github.com/unitaryfund/mitiq/assets/12703123/cf5b486b-9342-41c1-9468-85b617caa48d">

Reply:
Added suggestion to this effect.